.github/workflows/ci.yaml

```diff
@@ -58,6 +58,10 @@ jobs:
       - name: Install CRDs
         if: steps.list-changed.outputs.changed == 'true'
         run: make crds
+      
+      - name: Check installed crds
+        if: steps.list-changed.outputs.changed == 'true'
+        run: kubectl get crds
```

Comment:
?

Reply:
this was a test branch to debug the issue, please check out https://github.com/Nextdoor/k8s-charts/pull/140 instead